lib/utils/dataflow/src/index.js

```diff
@@ -715,11 +735,11 @@ const mapper = (mapfn, opt) => {
       return {
         body: dbclient.undbify(doc)
       };
-    }));
+    });
 
   // Retrieve error docs
   if(opt.error && opt.error.get)
-    routes.get(opt.error.get, throttle(function *(req) {
```

Comment:
@hsiliev 
+1 for that.
Outgoing requests must be throttled and incoming requests must be rate limited.